### PR TITLE
refactor: Remove assembly and output debug output to console when running UTs for java client

### PR DESCRIPTION
**Pull request by Apache9**
https://github.com/apache/incubator-pegasus/issues/1265

### CODE REVIEW

**Review by acelyc111 — APPROVED** [review]

File: java-client/configuration/log4j.properties [hunk 1]
```diff
@@ -1,42 +0,0 @@
-#
```
[the remark this answers is not in the document]
Comment (Apache9) [reply java-client/configuration/log4j.properties:1]: In java world we use maven or gradle to download dependencies, usually a jar and its transitive dependencies, and the configuration file should be prepared by the end users, when setting up its own application which uses pegasus java client.

And for running UTs, I moved the log4j.properties and pegasus.properties to src/test/resources. This is the default place for putting test resources, and you can load these fiels with getResourceAsStream when running UTs.

Thanks.

File: java-client/src/test/resources/log4j.properties [hunk 2]
```diff
@@ -17,15 +17,9 @@
...
-async_workers = 4
-enable_perf_counter = false
-perf_counter_tags = cluster=onebox,app=unit_test
-push_counter_interval_secs = 10
-meta_query_timeout = 5000
-auth_protocol =
-kerberos_service_name =
-kerberos_service_fqdn =
-kerberos_keytab =
-kerberos_principal =
```
Comment (empiredan) [inline java-client/src/test/resources/log4j.properties:31]: Is it necessary to also move these properties to `pegasus.properties` under `src/test/resources` as a configuration example for developers ?
```properties
meta_query_timeout = 5000
auth_protocol =
kerberos_service_name =
kerberos_service_fqdn =
kerberos_keytab =
kerberos_principal =
```

Comment (Apache9) [reply java-client/src/test/resources/log4j.properties]: Done.

**Review by empiredan — APPROVED** [review]

**Review by acelyc111 — APPROVED** [review]

File: java-client/configuration/log4j.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (levy5307) [reply java-client/configuration/log4j.properties:1]: OK. Thanks for your reply.

**Pull request merged**